src/v0/destinations/braze/README.md

```diff
@@ -0,0 +1,240 @@
+
+#### Custom Attribute Operations
+
+- Supports various operations on custom attributes:
+  - Remove
+  - Update
+  - Add
+  - Create
+
+## General Queries
+
+### Event Ordering
+
+- **Required**: No specific ordering requirements documented
+- Braze processes events based on the timestamp provided
+
+### Data Replay Feasibility
+
+#### Missing Data Replay
+
+- **Feasible**: Yes, for initial data loading
+- Braze explicitly supports importing legacy user data through the `/users/track` endpoint
+- Events are processed based on the provided timestamp
+- According to Braze documentation: "You may submit data through the Braze API for a user who has not yet used your mobile app to generate a user profile. If the user subsequently uses the application all information following their identification using the SDK will be merged with the existing user profile you created using the API call."
+
+#### Already Delivered Data Replay
+
+- **Partially Feasible**: Depends on the message type and configuration
+- **Identify Events**:
+
+  - With RudderStack deduplication enabled, duplicate user attributes will be filtered out, preventing duplicate profiles
+  - However, this doesn't leverage Braze's merge functionality, which requires explicit calls to `/users/identify` or `/users/merge` endpoints
+  - For proper identity resolution during replay, consider implementing custom logic to call these endpoints
+
+- **Track/Page/Screen Events**:
+
+  - **Not Recommended**: Braze treats each event as unique, even if it appears to be a duplicate
+  - According to Braze: "Each event object in the events array represents a single occurrence of a custom event by a user at a designated time. This means each event ingested into Braze has its own event ID, so 'duplicate' events are treated as separate, unique events."
+  - Replaying these events will create duplicates in Braze, potentially skewing analytics and triggering campaigns multiple times
+
+- **Purchase Events**:
+
+  - **Not Recommended**: Similar to track events, purchase events will be duplicated in Braze
+  - This could result in incorrect revenue calculations and purchase counts
+
+- **Configuration Considerations**:
+  - For user attribute updates only, enable RudderStack deduplication (`supportDedup` setting)
+  - For complete data replay including events, be aware that this will create duplicate events in Braze
+  - There is no native way to prevent event duplication in Braze during replay
+
+## Version Information
+
+### Current Version
+
+- Current implementation is in v0 (JavaScript)
+- Uses Braze REST API endpoints for data delivery
+- Supports both synchronous and asynchronous operations
+- No documented end-of-life date for this implementation
+
+### New Version Availability
+
+- No newer version documented at this time
+- Braze regularly updates their REST API, but the core endpoints used by this integration remain stable
+
+## Documentation Links
```

Comment:
Lets add a talking point on event ordering

Reply:
It's already here - https://github.com/rudderlabs/rudder-transformer/pull/4297/files?diff=unified&w=1#diff-6b0ee7f5d8b60d2ca97a18a529192b80613d61b597552007cb951033a67399d9R171